companion/src/firmwares/moduledata.h

```diff
@@ -135,7 +135,14 @@ enum MultiModuleRFProtocols {
   MODULE_SUBTYPE_MULTI_DSM_RX,
   MODULE_SUBTYPE_MULTI_JJRC345,
   MODULE_SUBTYPE_MULTI_Q90C,
-  MODULE_SUBTYPE_MULTI_LAST = MODULE_SUBTYPE_MULTI_Q90C
+  MODULE_SUBTYPE_MULTI_KYOSHO,
+  MODULE_SUBTYPE_MULTI_RLINK,
+  MODULE_SUBTYPE_MULTI_ELRS,
```

Comment:
@MRC3742, sorry for the necromancy, but could you please share what the idea for `MODULE_SUBTYPE_MULTI_ELRS` was? Surely this isn't ExpressLRS, right?

Reply:
Yes, ELRS was assigned a protocol slot in multi software and was going to be implemented with an additional add on rfm95w module.
https://github.com/pascallanger/DIY-Multiprotocol-TX-Module/pull/404

This PR was CLOSED and never implemented as ELRS prebuilt modules became widely available.

Rich